cosmos/carbon.json

```diff
@@ -110,9 +110,9 @@
       "coinDecimals": 18,
       "coinGeckoId": "binancecoin",
       "gasPriceStep": {
-        "low": 769230769.2307693,
-        "average": 961538461.5384616,
-        "high": 1153846153.8461537
+        "low": 1000000,
+        "average": 1000000,
+        "high": 11000000
```

Comment:
seems to be 100000? can doublecheck all values again?
![image](https://github.com/chainapsis/keplr-chain-registry/assets/7552714/a1aeace4-5984-4d20-a155-a172cdfe0a6b)